moveit_experimental/moveit_jog_arm/include/moveit_jog_arm/jog_calcs.h

```diff
@@ -89,7 +89,12 @@ class JogCalcs
 
   void publishWarning(bool active) const;
 
-  bool checkIfJointsWithinURDFBounds(trajectory_msgs::JointTrajectory_<std::allocator<void>>& new_joint_traj);
+  // Checks the current robot state for compliance with joint limits.
+  // If out of bounds, returns a halt trajectory that will stop the robot
+  // @param halting_joint_traj - the trajectory to execute to stop the robot if joints are out of bounds
+  // @return halting - false if robot is not within bounds and needs to hault
+  bool checkIfJointsWithinURDFBoundsAndCreateHaltTrajectory(
```

Comment:
Function name is getting long, perhaps rename to checkJointsInSRDFBoundsAndCreateHaltTrajectory(). That's a little better

Reply:
done